.github/workflows/publish.yml

```diff
@@ -5,18 +5,17 @@ on:
     types: closed
     branches: master
     paths: 'libs/**'
-
+    
 defaults:
   run:
     shell: bash
 
 jobs:
-  bump_version_and_publish:
-    name: Bump and publish new package version to npm
+  bump_package_versions:
```

Comment:
Perhaps throw in a little comment for the future describing that if core ischanged both core and designsystem is bumped. But if only designsystem is changed only that package is bumped. It is evident from the code - but might as well ease the mental load a bit.